api/lib/infrastructure/repositories/certification-result-repository.js

```diff
@@ -49,11 +57,31 @@ function _selectCertificationResults() {
       knex.raw(`
         json_agg("competence-marks".* ORDER BY "competence-marks"."competence_code" asc)  as "competenceMarks"`)
     )
+    .select(
+      knex.raw(`
+        COALESCE(json_agg("partner-certifications".*) filter (where "partner-certifications"."partnerKey" is not null), '[]') as "partnerCertifications"`)
```

Comment:
Pour simplifier la requete, on peut retirer le coalesce et faire un filter sur la liste

```suggestion
        json_agg("partner-certifications".*) as "partnerCertifications"`)
        
        ...
        
       const partnerCertifications = certificationResultDTO.partnerCertifications
      .filter(Boolean)
      .map((partnerCertification) => new PartnerCertification(partnerCertification));
```

Reply:
Done